archive/issues_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"testing"
 
 	"github.com/containerd/containerd/archive/compression"
-	"github.com/containerd/containerd/testutil"
+	"github.com/containerd/containerd/pkg/testutil"
```

Comment:
if this is just for testing, have you considered putting it in `internal`?

Reply:
I think most of the packages under `pkg` could be considered `internal`, but not sure that package naming is worth using.

Reply:
Yeah, I know we moved some packages to `internal` in the Moby to more clearly indicate we don't provide any compatibility guarantee (`pkg` feels more as "here, you can use this in your own project")

Reply:
we document what is supported and what is not.  things we expect ppl to depend on a broken out into separate repos.  I don't think its worth it at this point

Reply:
:+1: SGTM